schema/method_units/physical/psp/file/origin.json

```diff
@@ -0,0 +1,45 @@
+{
+    "schemaId": "method-physical-psp-file-origin",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "description": "Pseudopotential origin schema",
+    "properties": {
+        "slug": {
+            "description": "Slugified name of source",
+            "$ref": "#/definitions/ppSources"
+        },
+        "version": {
+            "description": "Version of source",
+            "type": "string"
+        },
+        "reference": {
+            "description": "Literature reference",
+            "$ref": "../../../../core/reference.json"
+        },
+        "comment": {
+            "description": "Brief description or other text annotation",
+            "type": "string",
+            "maxLength": 140
+        },
+        "generationDetails": {
+            "$ref": "generationDetails.json"
+        }
+    },
+    "definitions": {
+        "ppSources": {
+            "description": "",
+            "enum": [
+                "quantum espresso database",
+                "gbrv",
+                "vasp database",
```

Comment:
Same as above - just `vasp` would be sufficient, probably. The pseudopotentials are tied to a specific version of the code also - as long as the above `version` keyword tracks it already, we should be OK.